### DISCUSSION TITLE
Support for HT-A3000

### DISCUSSION THREAD

**Discussion by connorhawkins** [kovalev-sergey/homebridge-sony-audio#35] [Ideas]
I see there is support for the similar models of the HT-A Sound bars, so I am just curious if this model is able to be supported.

**Comment by kovalev-sergey** [comment]
All supported devices can be found [here](https://developer.sony.com/develop/audio-control-api/documentation/supported-devices/).
I wanted to say that Sony might add new ones, but was surprised to find on the official website that they no longer support the Audio Control API program. This means that support for new devices is not worth waiting for.

**Reply by connorhawkins** [reply]
Dang, that is discouraging. Thanks for the heads up!
